frontend/test/__support__/e2e/helpers/e2e-downloads-helpers.js

```diff
@@ -0,0 +1,74 @@
+const xlsx = require("xlsx");
+
+/**
+ * Trigger the download of CSV or XLSX files and assert on the results in the related sheet.
+ * It applies to both unsaved questions (queries) and the saved ones.
+ *
+ * @param {Object} params
+ * @param {("csv"|"xlsx")} params.fileType
+ * @param {number} [params.questionId]
+ * @param {function} callback
+ */
+export function downloadAndAssert(
+  { fileType, questionId, raw, logResults } = {},
```

Comment:
I forgot to document `raw` and `logResults`. Those can maybe even belong to a separate object `workbookOptions`?

Until we have the need to introduce more parameters to this function, it's probably ok to keep its structure like this, imo.